common/djangoapps/student/migrations/0018_delete_password_history.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-12-11 18:43
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
Are you completely sure that the `PasswordHistory` table isn't being accessed at all in production?  Is `ADVANCED_SECURITY_CONFIG` set to `False` in all of our production environments?

If not, we will need to make the database migration in a separate PR/release from the rest of the PR, given our Blue-Green deployments.  See [Deployment and backward-compatible migrations](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-Deploymentandbackward-compatiblemigrations).

Reply:
Yes! You are completely right. I need to do a separate PR. Will retool later.

Reply:
Done. I've dropped the migration and restored the model class (minus its class methods).